### PR TITLE
Changed README file.

### PR DESCRIPTION
**Pull request by chen7572**

### CODE REVIEW

**Comment by ZachGlassman** [comment]
addresses https://github.com/thedataincubator/stockpredict/issues/7

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # stockpredict [![Build Status](https://travis-ci.org/thedataincubator/stockpredict.svg?branch=master)](https://travis-ci.org/thedataincubator/stockpredict)
 
+Overview:
+This repository contains a stock price predict app that is based on stock market data from Quandl. The goal of this repository is to work as a team to make some changes to the app and get experience on working on production codes. 
+
+Repository Organization:
+static: contains Google stock price file GOOGL_data.txt
```
Comment (ZachGlassman) [inline README.md]: lets change this to static resources in general.

Comment (chen7572) [reply README.md]: will do!

**Review by ZachGlassman — APPROVED** [review]

**Comment by ZachGlassman** [comment]
LGTM, merge away

**Comment by adrianyi** [comment]
It comes out as a giant paragraph, need to add line breaks.

**Review by adrianyi — APPROVED** [review]
Looks good!

**Pull request merged**